svf/include/SVFIR/SVFValue.h

```diff
@@ -324,7 +324,7 @@ class SVFFunction : public SVFValue
     BasicBlockGraph* bbGraph; /// the basic block graph of this function
 
 protected:
-    inline void setCallGraphNode(CallGraphNode *cgn)
+    inline void setCallGraphNode(const CallGraphNode *cgn)
```

Comment:
use https://github.com/SVF-tools/SVF/pull/1646/files#diff-6873579e48857d104dff081895d5663ba73f00e17e370ac6c2304b02b8e4940eL58 to get callgraphnode from function